cmd/image-inspector.go

```diff
@@ -28,6 +29,8 @@ func main() {
 
 	flag.Parse()
 
+	inspectorOptions.AuthToken = os.Getenv("INSPECTOR_AUTH_TOKEN")
```

Comment:
do we want to make this consistent with post-results-token-file? i mean should this be a token inside file instead of env var?